### DISCUSSION TITLE
Lian Li TL Fans

### DISCUSSION THREAD

**Discussion by villainous-j** [Rem0o/FanControl.Releases#2624] [General]
Has anyone had any luck with getting these fans to work with Fan Control? The [Lian Li plugin](https://github.com/EightB1ts/FanControl.LianLi) doesn't seem to detect them, and I've even taken a stab at adding in the new Vendor and Product ID's to that same plugin with no success.

L-Connect is such trash that I'd love to get these fans integrated back into Fan Control like I've always run my case fans!

**Comment by Coldblackice** [comment]
Did you have any luck on this?

**Reply by villainous-j** [reply]
i wish!

**Comment by arishokri** [comment]
Desperately needed. I cannot control my Lian-Li AIO pump or fan speed.
